### PR TITLE
refactor(crosswalk_traffic_light_estimator)!: add autoware_ prefix

### PR DESCRIPTION
**Pull request by yuki-takagi-66**
## Description
Add `autoware_` prefix to `crosswalk_traffic_light_estimator`.

Part of:
- https://github.com/autowarefoundation/autoware/issues/4569

## Related links
https://github.com/autowarefoundation/autoware/issues/4569

## Tests performed
works well in psim

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

**Review by satoshi-ota — APPROVED** [review]

File: perception/autoware_crosswalk_traffic_light_estimator/CMakeLists.txt [hunk 1]
```diff
@@ -10,12 +10,12 @@ autoware_package()
...
 
-ament_auto_add_library(crosswalk_traffic_light_estimator SHARED
+ament_auto_add_library(autoware_crosswalk_traffic_light_estimator SHARED
   src/node.cpp
 )
 
-rclcpp_components_register_node(crosswalk_traffic_light_estimator
-  PLUGIN "traffic_light::CrosswalkTrafficLightEstimatorNode"
+rclcpp_components_register_node(autoware_crosswalk_traffic_light_estimator
+  PLUGIN "autoware_crosswalk_traffic_light_estimator::CrosswalkTrafficLightEstimatorNode"
```
Comment (satoshi-ota) [inline perception/autoware_crosswalk_traffic_light_estimator/CMakeLists.txt:18]:
```suggestion
  PLUGIN "autoware::crosswalk_traffic_light_estimator::CrosswalkTrafficLightEstimatorNode"
```

**Review by satoshi-ota — CHANGES_REQUESTED** [review]

File: perception/autoware_crosswalk_traffic_light_estimator/include/crosswalk_traffic_light_estimator/node.hpp [hunk 2]
```diff
@@ -117,6 +117,6 @@ class CrosswalkTrafficLightEstimatorNode : public rclcpp::Node
   std::shared_ptr<DebugPublisher> pub_processing_time_;
 };
 
-}  // namespace traffic_light
+}  // napespace autoware::crosswalk_traffic_light_estimator
```
Comment (satoshi-ota) [inline perception/autoware_crosswalk_traffic_light_estimator/include/crosswalk_traffic_light_estimator/node.hpp:120]: typo

**Review by satoshi-ota — CHANGES_REQUESTED** [review]

File: perception/autoware_crosswalk_traffic_light_estimator/CMakeLists.txt (hunk 1 above)
Comment (yuki-takagi-66) [reply perception/autoware_crosswalk_traffic_light_estimator/CMakeLists.txt]: Thank you. I've fixed.

File: perception/autoware_crosswalk_traffic_light_estimator/include/crosswalk_traffic_light_estimator/node.hpp (hunk 2 above)
Comment (yuki-takagi-66) [reply perception/autoware_crosswalk_traffic_light_estimator/include/crosswalk_traffic_light_estimator/node.hpp]: Thank you. I've fixed.

**Review by satoshi-ota — APPROVED** [review]

**Pull request merged**